modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -0,0 +1,71 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distereoibution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_STEREO_HPP
+#define OPENCV_GAPI_STEREO_HPP
+
+#include <opencv2/gapi/gmat.hpp>
+#include <opencv2/gapi/gscalar.hpp>
+#include <opencv2/gapi/gkernel.hpp>
+
+namespace cv {
+namespace gapi {
+namespace stereo {
+
+G_TYPED_KERNEL(GStereoBM, <GMat(GMat, GMat, const bool, const int, const int,
```

Comment:
Why `GStereoBM`?
We should have just `Stereo` operation, where `StereoBM` specifies that this operation will be implemented using Block Matching algorithm for correspondance computation. 
I think such information should be hided in implementation details(kernel stage).

Reply:
Fixed